packages/flutter/lib/src/painting/colors.dart

```diff
@@ -457,6 +457,44 @@ class ColorSwatch<T> extends Color {
 
   @override
   String toString() => '${objectRuntimeType(this, 'ColorSwatch')}(primary value: ${super.toString()})';
+
+  /// Linearly interpolate between two [ColorSwatch]es.
+  ///
+  /// It delegates in [Color.lerp] to interpolate the different colors of the
```

Comment:
Nit: If I'm reading this right, I think this makes more sense: "in" should be "to"